src/test/java/site/onandoff/util/encryption/BcryptManagerTest.java

```diff
@@ -0,0 +1,75 @@
+package site.onandoff.util.encryption;
+
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class BcryptManagerTest {
+
+	@Test
+	@DisplayName("평문을 60자의 해쉬 암호로 만들어 반환한다.")
+	void encrypt() {
+		// given
+		String plainPassword1 = "hi";
+		String plainPassword2 = "hello";
+		String plainPassword3 = "안녕";
+
+		// when
+		String hashPassword1 = BcryptManager.encrypt(plainPassword1);
+		String hashPassword2 = BcryptManager.encrypt(plainPassword2);
+		String hashPassword3 = BcryptManager.encrypt(plainPassword3);
+
+		// then
+		assertThat(hashPassword1.length()).isEqualTo(60);
+		assertThat(hashPassword2.length()).isEqualTo(60);
+		assertThat(hashPassword3.length()).isEqualTo(60);
+	}
+
+	@Test
+	@DisplayName("같은 평문을 암호화하더라도 결과는 항상 다르다.")
+	void sameInputDifferentOutput() {
+		// given
+		String plainPassword1 = "hi";
+		String plainPassword2 = "hi";
+		String plainPassword3 = "hi";
+
+		// when
+		String hashPassword1 = BcryptManager.encrypt(plainPassword1);
+		String hashPassword2 = BcryptManager.encrypt(plainPassword2);
+		String hashPassword3 = BcryptManager.encrypt(plainPassword3);
```

Comment:
💁🏻 이렇게 줄일 수도 있을 것 같아요
```suggestion
		// given
		String plainPassword = "hi";

		// when
		String hashPassword1 = BcryptManager.encrypt(plainPassword);
		String hashPassword2 = BcryptManager.encrypt(plainPassword);
		String hashPassword3 = BcryptManager.encrypt(plainPassword);
```

Reply:
입력한 평문 패스워드가 각각 다른 유저들의 평문 패스워드 입력임을 강조하고 싶었어요.